docs/openfaas-cloud/self-hosted/secrets.md

```diff
@@ -30,86 +30,4 @@ The preferred method available for Kubernetes uses SealedSecrets by Bitnami. We
-6. Reference the secret in `stack.yml` without your username this time i.e. `- hallo` - note this is the name of the repo without your username prefixed. The prefix will be added automatically.
-7. Install the GitHub App if you haven't done this already.
-8. Do a Git push to trigger a build
-9. Await the success status on the commit then click it to follow through to the live link
-
-View your overview page at: `https://system.DOMAIN-NAME.TLD/dashboard/username`
-
-##### Example [from reference repository](https://github.com/alexellis/my-fn):
-
-We'll encrypt an incoming webhook URL for Slack, which should be considered as confidential information.
-
-Let's write the example function:
-
-```yaml
-def handle(req):
-    webhook_url = None
-    with open("/var/openfaas/secrets/incoming-webhook-url") as webhook_url_text:
-        webhook_url = webhook_url_text.read().strip()
-
-    respond_to_user(req, webhook_url)
-```
-*handler.py*
-
-We see that the secret is read as per any other OpenFaaS secret, from the mounted location on disk at: `/var/openfaas/secrets/`.
-
-Now let's look at how we seal the secret:
-
-```sh
-# Seal secrets for owner alexellis named `fn-secrets`
-
-faas-cli cloud seal --name alexellis-fn-secrets --literal incoming-webhook-url=https://...
-```
-
-Here's the file generated by the command above:
-
-```yaml
-apiVersion: bitnami.com/v1alpha1
-kind: SealedSecret
-metadata:
-  creationTimestamp: null
-  name: alexellis-fn-secrets
-  namespace: openfaas-fn
-spec:
-  encryptedData:
-    incoming-webhook-url: (redacted)
-```
-*secrets.yaml*
-
-And finally, we now need to reference the name of our secret in `stack.yml`. Notice that the key and the secret name do not have to match, this is because we can have multiple secret key/values within a single secret.
-
-```yaml
-provider:
-  name: faas
-  gateway: http://127.0.0.1:8080
-
-functions:
-  slack-me:
-    lang: python
-    handler: ./slack-me
-    image: alexellis/slack-me
-    secrets:
-      - fn-secrets
-```
-*stack.yml*
-
-#### Troubleshooting
-
-The steps above must be followed precisely and if you have mis-read any of the details this may result in the secret not being accessible.
-
-Notes:
-
-* When using `kubeseal` your secret name needs to be prefixed with your username i.e. `alexellis-my-secret`
-* Each `--literal` must have no prefix, it's the exact name you will use in `stack.yml`
-* In `stack.yml` your secrets should have no prefix, this is added later automatically
-* You must commit `secrets.yaml` into your repo and do a `git push`
-
-If in doubt check your results against [this reference repository](https://github.com/alexellis/my-fn).
-
+See the community [secrets](../secrets.md) documentation for information on how to use sealed secrets. 
```

Comment:
Not specifically about "community". I'd remove

Reply:
👍